File: src/test/java/fi/metatavu/metaform/server/test/functional/FileUploadMeta.java
package fi.metatavu.metaform.server.test.functional;

/**
 * Model for uploaded file meta
 *
 * @author Antti Leppä
 */
public class FileUploadMeta {

  private String contentType;
  private String fileName;

  public String getFileName() {
    return fileName;
  }

  public void setFileName(String fileName) {
    this.fileName = fileName;
  }

  public String getContentType() {
    return contentType;
  }

  public void setContentType(String contentType) {
    this.contentType = contentType;
  }

} 

File: src/test/java/fi/metatavu/metaform/server/test/functional/FileUploadMeta.kt
package fi.metatavu.metaform.server.test.functional

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * Model for uploaded file meta
 *
 * @author Antti Leppä
 */
class FileUploadMeta (
    @JsonProperty("contentType")
    val contentType: String,
    @JsonProperty("fileName")
    val fileName: String
)